control-station/src/services/PodSocketClient.ts

```diff
@@ -114,6 +120,25 @@ class PodSocketClient {
 	private onData(data: string): void {
 		console.log("server says", data);
 	}
+
+	private addMessage(response: string, newState: State): void {
+		const currentTime = new Date();
+		const hours = currentTime.getHours().toString().padStart(2, "0");
+		const minutes = currentTime.getMinutes().toString().padStart(2, "0");
+		const seconds = currentTime.getSeconds().toString().padStart(2, "0");
+		const timestamp = `${hours}:${minutes}:${seconds}`;
```

Comment:
Suggestion: rather than reimplementing the formatting ourselves, we could use [`Date.toLocaleTimeString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString)
```suggestion
		const timestamp = currentTime.toLocaleTimeString("en-US", { hour12: false });
```